content/agent/kubernetes/host_setup.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Kubernetes Host Setup
+kind: documentation
+further_reading:
+- link: "agent/autodiscovery"
+  tag: "Documentation"
+  text: Docker Agent Autodiscovery
+- link: "agent/kubernetes/host_setup"
+  tag: "documentation"
+  text: "Kubernetes Host Setup"
+- link: "agent/kubernetes/integrations"
+  tag: "documentation"
+  text: "Custom Integrations"
+---
+
+Installing the Agent directly on your host (rather than having the Agent run in a Pod) provides additional visibility into your ecosystem, independent of Kubernetes.
+
+To gather your kube-state metrics:
+
+1. Download the [Kube-State manifests folder][3].
+
+2. Apply them to your Kubernetes cluster:
+  ```
+  kubectl apply -f <NAME_OF_THE_KUBE_STATE_MANIFESTS_FOLDER>
+  ```
+
+## Setup Kubernetes DNS
```

Comment:
kube-dns is the name of this piece of software, it's better to spell it `kube-dns`, `Kubernetes DNS` can be confused with something else.

Also it should be `Setup the kube-dns integration`. We don't provide DNS for Kubernetes, we only monitor it.